pizzeria-backend/Hubs/OrderHub.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Newtonsoft.Json;
+using pizzeria_backend.Models;
+
+namespace pizzeria_backend.Hubs
+{
+
+    public interface IOrderHub
+    {
+        public Task SendMessage(string user, string message);
```

Comment:
this SendMessage sounds like something generic you used for testing and you should delete now

Reply:
Done